noir/noir-repo/compiler/noirc_frontend/src/hir/def_collector/dc_mod.rs

```diff
@@ -41,16 +42,28 @@ pub fn collect_defs(
     module_id: LocalModuleId,
     crate_id: CrateId,
     context: &mut Context,
+    macro_processors: &Vec<&dyn MacroProcessor>,
```

Comment:
Same as above!

Reply:
In general, just run clippy manually because in aztec-packages clippy is not run for noir

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/5101

Reply:
Done, makes sense I had to run it manually